From: SeongJae Park <sj@kernel.org>
Date: Sat, 21 Jun 2025 14:27:09 -0700
Subject: [PATCH] ==== auto-tune trace events ====

mm/damon: add trace events for auto-tuned monitoring intervals and DAMOS quota

The aim-oriented auto-tuning features for monitoring intervals and DAMOS
quota are important and recommended.  Add tracepoints for observabilities
of those tuned values and the tuning itself.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_vstl_jfh | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_vstl_jfh

diff --git a/hkml_cv_bogus/hkml_cv_bogus_vstl_jfh b/hkml_cv_bogus/hkml_cv_bogus_vstl_jfh
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

